scripts/lib/CIME/SystemTests/err.py

```diff
@@ -29,3 +30,12 @@ def _case_two_custom_prerun_action(self):
         expect(len(restart_list) >= 1, "No restart files found in {}".format(rest_root))
         restore_from_archive(self._case, rest_dir=
             os.path.join(rest_root, restart_list[0]))
+
+    def _case_two_custom_postrun_action(self):
+        # Link back to original case1 name
+        for case_file in glob.iglob(os.path.join(self._case1.get_value("RUNDIR"),
+                                                 "*.nc.{}".format(self._run_one_suffix))):
+            orig_file = case_file[:-(1+len(self._run_one_suffix))]
+            if not os.path.isfile(orig_file):
+                os.symlink(case_file, orig_file)
```

Comment:
Will a symlink work robustly if you try to rerun a test? My head is spinning trying to think this through, but I could foresee possible problems. A copy seems safer in case the test gets rerun, even though it's a bit more expensive.